watertap/property_models/seawater_prop_pack.py

```diff
@@ -1500,10 +1496,8 @@ def _therm_cond_phase(self):
             units=pyunits.W / pyunits.m / pyunits.K,
             doc="Thermal conductivity",
         )
-
-        def rule_therm_cond_phase(
-            b, p
-        ):  # thermal conductivity, eq. 13 in Sharqawy  et al. (2010)
+        # Sharqawy  et al. (2010), eq. 13, 0-180 C, 0-160 g/kg
```

Comment:
It would be nice if we could show logger output, say, after a model was solved, to display warnings for any variables out of range. Just a thought--not something that I think should be in this PR if we did have some form of an implementation in mind.